parts/k8s/kubernetesinstalls.sh

```diff
@@ -169,7 +169,7 @@ installClearContainersRuntime() {
 }
 
 installNetworkPlugin() {
-    if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
+    if [[ "${NETWORK_PLUGIN}" = "azure" ]] || [[ "${NETWORK_PLUGIN}" = "calico+azureipam"  ]]; then
```

Comment:
what is calico+azureipam? I may be missing something here. Can you please explain us which scenario this is used?